subsystems/mvdm/ntvdm/vddsup.c

```diff
@@ -341,7 +341,7 @@ static BOOL LoadInstallableVDD(VOID)
     VDDValueName = VDDList;
     while (*VDDList)
     {
-        DPRINT1("Loading VDD '%S'...", VDDList);
+        DPRINT1("Loading VDD '%S'... ", VDDList);
```

Comment:
```suggestion
        DPRINT1("Loading VDD '%S'...\n", VDDList);
```

Reply:
No, `\n` is unwanted here.

Reply:
Please revert. You should have understood the code that appends "Failed" or "Succeeded" with a newline accordingly just after that string.

Reply:
Resolved.

Reply:
The extra space is okay for the master stream.